public/app/features/plugins/pluginPreloader.ts

```diff
@@ -0,0 +1,13 @@
+import { importPluginModule } from './plugin_loader';
+
+export async function preloadPlugins(pluginsToPreload: string[] = []): Promise<void> {
+  await Promise.all(pluginsToPreload.map(preloadPlugin));
+}
+
+async function preloadPlugin(path: string): Promise<void> {
+  try {
+    await importPluginModule(path);
+  } catch (error: any) {
```

Comment:
Try `error: unknown`  here for when you don't know what the type is and we can just treat it as an opaque value.

`unknown` is like `any` but spicy. It prevents people from making assumptions about what it is.

Reply:
Great feedback! I will fix that.

Regarding the error. We started with adding a toast error message about it failing. But you will get the same feedback when trying to use the plugin. So it feels like that is good enough.